### DISCUSSION TITLE
Turn off runOnDemand?

### DISCUSSION THREAD

**Discussion by totalitarian** [fuatakgun/eufy_security#391] [Q&A] [answered]
Hi, any way to turn off the runOnDemand feature?

```
RTSP_PATHS_ALL_RUNONDEMAND: echo 'incoming'
RTSP_PATHS_ALL_RUNONDEMANDRESTART: 'yes'
```

**Comment by fuatakgun** [comment, marked as answer]
According to my initial testing, this was helping to establish initial video stream faster way. Why would you prefer to do this?

**Reply by totalitarian** [reply]
I’ve changed my mind. The webRTC card was reloading the stream every time I switched to another tab and back. Turns out it was an issue with my card config.
